src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -61,9 +61,41 @@ trait MarathonConf extends ScallopConf with ZookeeperConf with IterativeOfferMat
   def executor: Executor = Executor.dispatch(defaultExecutor())
 
   lazy val mesosRole = opt[String]("mesos_role",
-    descr = "Mesos role for this framework",
+    descr = "Mesos role for this framework. " +
+      "If set, Marathon receives resource offers for the specified role in addition to " +
+      "resources with the role designation '*'.",
     default = None)
 
+  def expectedResourceRoles: Set[String] = mesosRole.get match {
+    case Some(role) => Set(role, "*")
+    case None       => Set("*")
+  }
+
+  lazy val defaultAcceptedResourceRolesSet = defaultAcceptedResourceRoles.get.get
+
+  lazy val defaultAcceptedResourceRoles = opt[String]("default_accepted_resource_roles",
+    descr =
+      "Default for the defaultAcceptedResourceRoles attribute of all app definitions" +
+        " as a comma-separated list of strings." +
+        "This defaults to all roles for which this Marathon instance is configured to receive offers.",
+    default = Some(expectedResourceRoles.mkString(",")),
```

Comment:
This default value leads to an exception while generating the usage with --help.
```
> run --help
[info] Running mesosphere.marathon.Main --help
[2015-05-18 16:11:59,605] INFO Starting Marathon 0.8.2-SNAPSHOT (mesosphere.marathon.Main$:87)
[error] (run-main-1) org.rogach.scallop.exceptions.UnknownOption: Unknown option 'help'
org.rogach.scallop.exceptions.UnknownOption: Unknown option 'help'
	at org.rogach.scallop.Scallop$$anonfun$5.apply(Scallop.scala:125)
	at org.rogach.scallop.Scallop$$anonfun$5.apply(Scallop.scala:125)
	at scala.Option.getOrElse(Option.scala:120)
	at org.rogach.scallop.Scallop.parse(Scallop.scala:124)
	at org.rogach.scallop.Scallop.parse(Scallop.scala:85)
	at org.rogach.scallop.Scallop.org$rogach$scallop$Scallop$$parsed$lzycompute(Scallop.scala:187)
	at org.rogach.scallop.Scallop.org$rogach$scallop$Scallop$$parsed(Scallop.scala:175)
	at org.rogach.scallop.Scallop$$anonfun$get$5.apply(Scallop.scala:540)
	at org.rogach.scallop.Scallop$$anonfun$get$5.apply(Scallop.scala:537)
	at scala.Option.map(Option.scala:145)
	at org.rogach.scallop.Scallop.get(Scallop.scala:537)
	at org.rogach.scallop.ScallopConf$$anon$1$$anonfun$fn$1.apply(ScallopConf.scala:126)
	at org.rogach.scallop.ScallopConf$$anon$1$$anonfun$fn$1.apply(ScallopConf.scala:126)
	at org.rogach.scallop.ScallopOption.get(ScallopOption.scala:25)
	at mesosphere.marathon.MarathonConf$class.expectedResourceRoles(MarathonConf.scala:69)
	at mesosphere.marathon.MarathonApp$AllConf.expectedResourceRoles(Main.scala:76)
	at mesosphere.marathon.MarathonConf$$anonfun$defaultAcceptedResourceRoles$1.apply(MarathonConf.scala:81)
	at mesosphere.marathon.MarathonConf$$anonfun$defaultAcceptedResourceRoles$1.apply(MarathonConf.scala:81)

```